README.md

```diff
@@ -2,6 +2,15 @@
 
 Discussion agents.
 
+# Windows installation 
+```
+1.First install anaconda https://docs.anaconda.com/free/anaconda/install/windows/ and add to path, or launch anaconda console
```

Comment:
Add a space after the number

Reply:
Also, the link can be formatted with markdown like this [anaconda](https://docs.anaconda.com/free/anaconda/install/windows/). End each step with a period (full sentences).

Reply:
Probably go into detail (alternatively, you can add a link to some resource for how to add stuff to the path) describing how to add to the path.